client/state/selectors/get-reader-aliased-follow-feed-url.js

```diff
@@ -1,13 +1,15 @@
 /**
  * External Dependencies
  */
-import { find, includes } from 'lodash';
+import { find, includes, some } from 'lodash';
 
 /**
  * Internal Dependencies
  */
 import { prepareComparableUrl } from 'state/reader/follows/utils';
 
+export const commonExtensions = [ 'rss', 'rss.xml', 'feed', 'atom/feed', 'atom.xml', 'atom' ];
```

Comment:
speed isn't really an issue here considering it'll need to exhaust the whole list 99% of the time and this runs only for the follow by url component